app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -41,22 +36,8 @@ class MainViewModel : ViewModel() {
     fun fetchWeatherLocations(locationName: String) {
         weatherLocations = weatherService.fetchWeatherByLocation(locationName)
     }
-    fun save(reminder: Reminder, user: FirebaseUser){
-        val document = firestore.collection("weather").document()
-        reminder.city = document.id
-        val set = document.set(reminder)
-        set.addOnSuccessListener {
-            Log.d("Firebase", "document saved")
-           /**
-            if (reminder != null && reminder.size > 0) {
-                saveReminders(reminder, user)
-            }
-            */
-        }
-        set.addOnFailureListener {
-            Log.d("Firebase", "Save Failed")
-        }
-
+    fun save(reminder: Reminder, user: FirebaseUser) {
+        firestore.save(reminder, user)
```

Comment:
Now calls save method from the FirestoreServicer object

Reply:
This was implement this week  look just like this.

Reply:
Great discussion!